lib/elixir/src/elixir.erl

```diff
@@ -10,7 +10,9 @@
 -define(system, 'Elixir.System').
 
 %% Top level types
--export_type([char_list/0, struct/0, as_boolean/1, keyword/0, keyword/1]).
+%% TODO: Deprecate in v1.5 - type char_list
```

Comment:
it does read much better. I just didn't want to break the "TODO: Deprecate in v1.5" so a simple search would list all changes for v1.5.
But if we agree on the change I can do it,